docs/index.rst

```diff
@@ -1,244 +1,61 @@
-.. django-otp-webauthn-doc documentation master file, created by
-   sphinx-quickstart on Thu Sep 26 15:47:57 2024.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+About Django OTP WebAuthn
+=========================
 
-Django OTP WebAuthn
-===================
+Django OTP WebAuthn is an implementation of `WebAuthn Passkeys <https://passkeys.dev>`_ for Django applications. It extends the `django-otp <https://github.com/django-otp/django-otp>`_ framework to support multi-factor authentication using :ref:`passkeys <about-passkeys>`.
 
-|Version| |License| |Build Status| |codecov| |All Contributors| |pre-commit|
-
-This package provides an implementation of `WebAuthn Passkeys <https://passkeys.dev/>`_ for Django. It is written as a plugin for the `Django OTP framework <https://github.com/django-otp/django-otp>`_ for multi-factor authentication. Under the hood, this package uses `py_webauth <https://github.com/duo-labs/py_webauthn/>`_ to handle all cryptographic operations.
-
-.. important::
-   This package is in development not yet thoroughly tested and documented. The API is subject to change. If you are interested in using this package, please star this repository to show your interest. This will help me prioritize development. If you are interested in contributing, please see the `DEVELOPMENT.md <DEVELOPMENT.md>`_ file.
-
-Compatibility
--------------
-
-- Django >= 4.2
-- Python >= 3.9
-- django-otp >= 1.4.0
-
-Browser Compatibility
-^^^^^^^^^^^^^^^^^^^^^
-
-Passkeys are supported in most modern browsers. Here is a list of browsers that support Passkeys:
-
-- Chrome 67+
-- Firefox 60+
-- Safari 13+
-- Microsoft Edge 18+
-
-For a complete list, see `caniuse.com/webauthn <https://caniuse.com/webauthn>`_.
+Django OTP WebAuthn simplifies :term:`passkey <passkey/discoverable credential>` authentication by handling all cryptographic operations through `py_webauthn <https://github.com/duo-labs/py_webauthn/>`_.
```

Comment:
@activus-d should the documentation index explicitly link to the getting started guide? I know it is in the sidebar, but just looking at this page the 'where to go next / call to action' is not very obvious.

<img width="1118" alt="image" src="https://github.com/user-attachments/assets/c75b5d36-6b90-42e9-be49-718418ea115e" />